benchmark_runner/jupyterlab/templates/summary_report/summary_report_widgets.py

```
@@ -165,8 +166,12 @@ def get_workload_filtered_data(self, workload: str, filter_field: dict = None):
         Located in this class for get_compared_ocp_versions
         @return:
         """
-        entries = self.summary_report.get_workload_data(index=f'{workload}-results', start_datetime=self.DEFAULT_START_DATE,
-                                                             end_datetime=self.DEFAULT_END_DATE)
+        try:
+            entries = self.summary_report.get_workload_data(index=f'{workload}-results', start_datetime=self.DEFAULT_START_DATE,
+                                                                 end_datetime=self.DEFAULT_END_DATE)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)